Automattic/wp-calypso
# Products: Do not import the entire productsValues object

This PR updates the `lib/products-values` library to not export an entire object again with all the modules, as they're already exported. It also updates the single instance in the code where we import the entire module to import only the function that we need.

While this doesn't offer a significant bundle size improvement (less than half a KB down compressed from each entry point), it's a good practice and may help reduce the bundle size over time as we further improve things around what we import and export.

#### Changes proposed in this Pull Request

* Products: Do not export and import the entire `productsValues` object.

#### Testing instructions

* Input `localStorage.setItem('debug', '*');` in your console.
* Purchase a plan on both Calypso local and in prod.
    * You may not use staging to test this as ad-tracking is disabled in staging
    * To test it locally, you must enable ad-tracking locally
* Verify that all the `record order` debug data is the same between local and prod
    * In particular verify `recordOrder: wpcomJetpackCartInfo`'s content
* Verify that checkout works as it did before, with no errors in the console.
